app/src/schemas/toolCallSchemas.ts

```diff
@@ -265,3 +284,92 @@ export function createAnthropicToolCall(): AnthropicToolCall {
     input: {},
   };
 }
+
+export const toolCallHeuristicSchema = z.object({
```

Comment:
can you add a description here? I'm not fully grokking what makes it a heuristic.